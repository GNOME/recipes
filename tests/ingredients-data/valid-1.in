1		apple	
2		banana	
3	box	nuts	
