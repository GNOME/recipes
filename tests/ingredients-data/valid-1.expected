AMOUNT 1.000000
UNIT 
NAME Apple

AMOUNT 2.000000
UNIT 
NAME Banana

AMOUNT 3.000000
UNIT box
NAME nuts

